src/components/select/Select.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from 'react';
+
+import Select from './Select';
+
+import type { ComponentStory, ComponentMeta } from '@storybook/react';
+
+export default {
+  title: 'Components/Select',
+  component: Select,
+  parameters: { layout: 'centered' },
+} as ComponentMeta<typeof Select>;
+
+const Template: ComponentStory<typeof Select> = ({ selected, onChange, ...args }) => {
+  const [_selected, setSelected] = useState(selected);
+
+  return <Select {...args} selected={_selected} onChange={setSelected} />;
+};
+
+export const Default = Template.bind({});
+Default.args = {
+  label: '',
+  className: 'min-w-48',
+  options: Array.from({ length: 12 }, (_, i) => ({ key: i.toString(), value: `Option ${i}` })),
+};
+
+export const Disabled = Template.bind({});
```

Comment:
Disabled state make sure:
1. cursuor should be `default`, not `pointer` (and not `not-allowed`).
2. Use opacity like `50%` or other values instead of change background colors.
3. No pointer events.